src/client/runner/automation/playback/select.js

```diff
@@ -233,7 +231,7 @@ export default function (el, options, runCallback) {
                 point = positionUtils.findCenter(topElement);
 
             automationSelectUtil.scrollElementByPoint(topElement, point);
-            point = automationSelectUtil.updatePointByScrollElement(topElement, point);
+            point     = automationSelectUtil.updatePointByScrollElement(topElement, point);
 
```

Comment:
Wrong alignment. Also may be worth to remove newline and align this assignment with two another that are below.